pipeline-steps/deploy-network.yaml

```diff
@@ -5,71 +5,27 @@ parameters:
   project: ''
   tfversion: ''
   tfvars: ''
-  terraformServiceConnection: ''
+  serviceConnection: ''
```

Comment:
it seems like this file isn't used anymore looking at the pipeline?

Reply:
Correct. Removed the file and tested the pipeline.